wp-content/themes/core/pcss/components/button/_solid.pcss

```diff
@@ -15,91 +15,58 @@
 .c-btn,
 a.c-btn {
 	@mixin button-global;
-
-	background-color: var(--color-primary);
-	border-bottom: none;
-	color: var(--color-white);
-	font-family: var(--font-family-calibre);
-	font-size: var(--font-size-1);
-	letter-spacing: 0.05em;
-	line-height: var(--line-height-base);
-	text-align: center;
-	text-transform: uppercase;
-	padding: 17px 24px;
-
-	&:visited {
-		color: var(--color-white);
-	}
-
-	&:hover,
-	&:focus,
-	&:active {
-		background-color: var(--color-link-hover);
-		color: var(--color-white);
-	}
-
-	@media (--viewport-medium) {
-		font-size: var(--font-size-2);
-		line-height: var(--line-height-base);
-		padding: 19px 24px;
-	}
-}
-
-/* -------------------------------------------------------------------------
- * Button: Modifier - Full Width
- * ------------------------------------------------------------------------- */
-
-.c-btn--full,
-a.c-btn--full {
-	width: 100%;
+	@mixin button-base;
+	@mixin button-solid;
 }
 
 /* -------------------------------------------------------------------------
- * Button: Modifier - Small
+ * Button: Modifier - Icon
  * ------------------------------------------------------------------------- */
 
-.c-btn--sm,
-a.c-btn--sm {
-	font-size: var(--font-size-1);
-	padding: 16px 18px;
+.c-btn--icon,
+a.c-btn--icon {
 
-	@media (--viewport-medium) {
-		font-size: var(--font-size-1);
+	&:after {
```

Comment:
Let's ask design how they would imagine longer text playing out for a button with an icon, I could see us needing to do a bit more of a robust implementation where the icon is absolutely positioned so it stays to the right and vertically centered.